src/gnatss/utilities/geo.py

```diff
@@ -3,13 +3,29 @@
 Geospatial utilities module
 """
 import numpy as np
-from nptyping import NDArray
+from nptyping import Float64, NDArray, Shape
 
 
 def _get_rotation_matrix(
     lat_org: float, lon_org: float, to_enu: bool = True
```

Comment:
More of a question, are type hints needed and/or recommended when a default value is used? E.g. for `to_enu: bool = True`. One could argue that the `True` implies the type?

Reply:
Yea, one could imply this, but it's better to be more explicit. Also, I can forsee using pydantic in the future possibly to do argument types validation, and one of the powerful feature is it will cast to the type specified, so it can still understand if user sends in text like "true" instead of the actual boolean.